model/project_aliases.go

```diff
@@ -71,22 +71,30 @@ type ProjectAlias struct {
 type ProjectAliases []ProjectAlias
 
 // FindAliasesForProject fetches all aliases for a given project
-func FindAliasesForProject(projectID string) ([]ProjectAlias, error) {
-	out := []ProjectAlias{}
+func FindAliasesForProject(projectID string, version string) ([]ProjectAlias, error) {
```

Comment:
Right now we use this for repo projects as well, because they store aliases the same way; however, repos don't have parser projects (because they're just a logical grouping of settings not tied to one specific project) so it now may make sense to have a FindAliasesForRepo that only queries the alias collection.